webknossos/examples/apply_merger_mode.py

```diff
@@ -0,0 +1,102 @@
+from calendar import day_abbr
+from pathlib import Path
+from typing import Tuple, cast
+
+import fastremap
+
+import webknossos as wk
+
+
+def main() -> None:
+
+    ####################################
+    # Opening a merger mode annotation #
+    ####################################
+
+    nml = wk.Annotation.download(
+        "https://webknossos.org/annotations/Explorational/6204d2cd010000db0003db91"
+    ).skeleton
+
+    ###############################################
+    # Download and open the corresponding dataset #
+    ###############################################
+
+    wk.Dataset.download(
+        "l4_sample_dev", "scalable_minds", path="testdata/l4_sample_dev"
+    )
+
+    dataset = wk.Dataset.open("testdata/l4_sample_dev")
+    in_layer = cast(wk.SegmentationLayer, dataset.get_layer("segmentation"))
+    in_mag1 = in_layer.get_mag("1")
+
+    ##############################
+    # Compute segment id mapping #
+    ##############################
+
+    segment_id_mapping = {}
+    for graph in nml.flattened_graphs():
+        base = None
+        for node in graph.nodes:
+            segment_id = in_mag1.read(
+                absolute_offset=node.position, size=(1, 1, 1)
+            ).item()
+            if base is None:
+                base = segment_id
+            segment_id_mapping[segment_id] = base
+
+    print(
+        f"Found {len(list(nml.flattened_graphs()))} segment id groups with {len(segment_id_mapping)} nodes"
+    )
+
+    ############################
+    # Creating an output layer #
+    ############################
+
+    out_layer = dataset.add_layer(
+        "segmentation_remapped",
+        wk.SEGMENTATION_CATEGORY,
+        dtype_per_layer=in_layer.dtype_per_layer,
+        largest_segment_id=in_layer.largest_segment_id,
+    )
+    out_mag1 = out_layer.add_mag("1", compress=True)
+    out_layer.bounding_box = in_layer.bounding_box
+
+    ###################
+    # Apply remapping #
+    ###################
+
+    def apply_mapping_for_chunk(args: Tuple[wk.View, wk.View, int]) -> None:
+        (in_view, out_view, _) = args
+        cube_data = in_view.read()[0]
+        # pylint: disable=c-extension-no-member
+        fastremap.remap(
+            cube_data,
+            segment_id_mapping,
+            preserve_missing_labels=True,
+            in_place=True,
+        )
+        out_view.write(cube_data)
+
+    in_mag1.for_zipped_chunks(apply_mapping_for_chunk, out_mag1)
+
+    ########################################
+    # Optionally, downsample and re-upload #
+    ########################################
+
+    # out_layer.downsample()
```

Comment:
@jstriebel is there a way to disable this code from running in the tests, but showing it in the docs?

Reply:
Yes, you can check if the env var `PYTEST_CURRENT_TEST` is set, see
https://github.com/scalableminds/webknossos-libs/blob/master/webknossos/examples/learned_segmenter.py#L87

To not clutter it this much in the examples, we might also add some magic comment, where the line is then removed before execution by `exec_main_and_get_vars`, but let's just add this black magic when those checks keep piling up :magic_wand: